ext/spl/spl_observer.c

```diff
@@ -256,12 +259,16 @@ static inline HashTable* spl_object_storage_debug_info(zend_object *obj) /* {{{
 	array_init(&storage);
 
 	ZEND_HASH_FOREACH_PTR(&intern->storage, element) {
-		md5str = php_spl_object_hash(&element->obj);
+		zval zv;
+		ZVAL_OBJ(&zv, element->obj);
+		md5str = php_spl_object_hash(&zv);
```

Comment:
Change php_spl_object_hash to accept zend_object?

Reply:
Done, was considering whether to propose that separately